services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/Persistence.kt

```diff
@@ -211,6 +206,22 @@ class HankeEntity(
         }
     }
 
+    fun alkuPvm(): ZonedDateTime? {
```

Comment:
These functions are never called outside tests. Do we really need them?

Reply:
Removed.